public/app/types/organization.ts

```diff
@@ -14,5 +14,5 @@ export interface OrganizationPreferences {
 export interface OrganizationState {
   organization: Organization;
   preferences: OrganizationPreferences;
-  starredDashboards: DashboardAcl[];
+  starredDashboards: Dashboard[];
```

Comment:
Not sure this should be part of the organization state as the starred dashboard is part of the user (starring is user action) so maybe it is better to be part of a user state (would like to move some of the contextSrv.user stuff to a user redux store)